libvast/vast/format/json.hpp

```diff
@@ -150,6 +154,19 @@ const char* reader<Selector>::name() const {
   return Selector::name();
 }
 
+template <class Selector>
+vast::system::report reader<Selector>::status() const {
+  using namespace std::string_literals;
+  uint64_t invalid_line = invalid_line_count_;
+  uint64_t unknown_layout = unknown_layout_count_;
+  invalid_line_count_ = 0;
+  unknown_layout_count_ = 0;
+  return {
+    {name() + ".invalid-line"s, invalid_line},
+    {name() + ".unknown_layout"s, unknown_layout},
```

Comment:
Shouldn't these be pluralized too?

Reply:
They aren't currently to stay in line with the PCAP reader.

Reply:
Okay.